### DISCUSSION TITLE
Doesn't seem to support high workloads or connections

### DISCUSSION THREAD

**Discussion by icolonder** [Ysurac/openmptcprouter#2674] [Q&A]
Make a connection in a in a medium company.

Join 2 lines using
**bbbr2**
**fullmesh**
_scheduler_:**default,BLEST**

With **glorytun** udp I had no problems with **UDP** video calls.

The slowness of the network is with **TCP**, maybe **shadowsocks** is buggy OR it doesn't support handling many connections. I tried with v2ray but it had more crashes

A calm day if it is fast. But a saturated day does not support.

I configured a vpn l2tp to the same vps, to make a difference if it was server problems and balance the two lines at peak times and without drops. My ping moves between 50-70ms to vps.

But with mptcprouter at peak times I can't stand the load.

**Comment by Ysurac** [comment]
I need number of connections and load average on both router and VPS to make an answer.
SQM is enabled or not ?
What is the architecture used ?
What is the version used ?

**Reply by icolonder** [reply]
VPS debian(11): 5.4.207-mptcp 0.1028
OMR : openmptcprouter v0.59.1-5.4 r0+16594-ce92de8c8c
Model：Raspberry Pi 4 Model B Rev 1.4
Architecture：ARMv8 Processor rev 3

SQM enabled
---
Mptcp connections established: 451

My Router connections show: 4380

**Comment by v1net** [comment]
maybe is could be a nice idea to use some more professional ethernet Hardware.

Is there someone who is using this router openmptcprouter with high load average? With more than 100 or 500 Clients? I will test it in the next Months. 

in the past i had a mini pc with load balancing and after arround 100 Clients the pc had a watchdog because of the ethernet packets and reboots.

**Reply by tianTian1312** [reply]
Hello, I encountered a phenomenon where the client starts to have a high delay when it exceeds 20.
I would like to ask if you have encountered a similar situation?
